Demo_1/index.html

```diff
@@ -0,0 +1,496 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Shop Design 1</title>
+
+    <script src="../dist/vendor/custom-elements-es5-adapter.js"></script>
+    <script src="../dist/vendor/webcomponents-loader.js"></script>
+
+    <link rel="stylesheet" href="css/asn.css" />
+    <link rel="stylesheet" href="css/header-navigation.css" />
+    <link rel="stylesheet" href="css/records.css">
+    <link rel="stylesheet" href="css/campaigns.css">
+    <link rel="stylesheet" href="css/spinner.css">
+    <link rel="stylesheet" href="css/tagcloud.css">
+    <link rel="stylesheet" href="css/breadcrumb.css">
+    <link rel="stylesheet" href="css/productDetail.css">
+    <link rel="stylesheet" href="css/sortbox.css">
+    <link rel="stylesheet" href="css/suggest.css">
```

Comment:
Why isn't folder structure of Demo_1 and Demo_2 organized the same way? We have additional folder 'resources' in Demo_2. I suggest making it consistent.